app/views/users/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Users</h1>
+
+<table class="table">
+  <thead>
+    <th>Email</th>
+    <th>First name</th>
+    <th>Last name</th>
+    <th>Last sign in</th>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td><%= link_to user.email, user_path(user) %></td>
+        <td><%= user.user_profile.first_name %></td>
+        <td><%= user.user_profile.last_name %></td>
+        <td><%= user.current_sign_in_at %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>`
```

Comment:
do we need to apply style guides for this page

Reply:
I will check, thanks!